e2e/_suites/fleet/fleet.go

```diff
@@ -1300,16 +1344,21 @@ func (fts *FleetTestSuite) thePolicyIsUpdatedToHaveSystemSet(name string, set st
 	if name != "log" {
 		condition = true
 	}
+	if name == "linux/metrics" {
+		condition = true
+		file = "/linux_metrics.json"
+		metrics = "linux"
+	}
 
 	if condition != true {
```

Comment:
I'd move this evaluation to the top of the method, to fail fast. Therefore, The above condition variable would not be needed.